app/code/Magento/InventoryConfiguration/Model/StockConfigurationInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryConfiguration\Model;
+
+interface StockConfigurationInterface
+{
+    public function validate($sku, $stockId, $qtyWithReservation, $isSalable) : bool;
```

Comment:
add types for paramers and PHP DocBlock describing them
`string $sku, int $stockId, float $qtyWithReservation, boolean $isSalable`

Reply:
Done